cluster/saltbase/salt/top.sls

```diff
@@ -33,6 +34,14 @@ base:
     - kube-controller-manager
     - kube-scheduler
     - monit
+    - contrail-apiserver
```

Comment:
Yes. Many of the hard-coded stuff need to be cleaned up. I wanted to give
Sanju a heads up.
10.0.0.0/16 is portal network in my aws setup.

Regards,
Ananth

On Tue, Aug 4, 2015 at 6:32 PM, Pedro Marques <notifications@github.com>
wrote:

> In cluster/saltbase/salt/top.sls
> <https://github.com/rombie/kubernetes/pull/1#discussion_r36262241>:
>
> > @@ -33,6 +34,14 @@ base:
> >      - kube-controller-manager
> >      - kube-scheduler
> >      - monit
> > +    - contrail-apiserver
>
> These will have to be conditional.d
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/rombie/kubernetes/pull/1/files#r36262241>.
>